fetch/content-length/resources/content-lengths.json

```diff
@@ -19,6 +19,18 @@
     "input": "Content-Length: 30",
     "output": 30
   },
+  {
+    "input": "Content-Length: 30,30",
+    "output": 30
+  },
+  {
+    "input": "Content-Length: 30\r\nContent-Length: 30",
+    "output": 30
+  },
+  {
+    "input": "Content-Length: 30\r\nContent-Length: 30,30",
+    "output": 30
+  },
```

Comment:
One other suggestion:  We could have a quoted test or two, to make sure quotes aren't handled:

Content-Length: "30"